app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -108,41 +103,24 @@ class AuditController @Inject() (
   private def fileId(): FileId =
     FileId(s"${UUID.randomUUID().toString}-${DateTimeFormatter.ofPattern("yyyyMMdd-HHmmss.SSS").withZone(ZoneOffset.UTC).format(Instant.now())}")
 
-  private def getSource(auditType: AuditType, uri: Option[ObjectStoreResourceLocation], request: Request[Source[ByteString, _]])(exceedsLimit: Boolean)(implicit
+  private def getSource(auditType: AuditType, request: Request[Source[ByteString, _]])(exceedsLimit: Boolean)(implicit
     hc: HeaderCarrier
   ): EitherT[Future, PresentationError, Payload] =
-    (uri, exceedsLimit) match {
-      case (None, false) =>
-        logger.info("Payload in body and < auditing message limit")
-        convertIfNecessary(auditType, request).asPresentation
-          .map(Right(_))
-      case (Some(uri), false) =>
-        logger.info("Payload in object store and < auditing message limit")
-        val contents = objectStoreService
-          .getContents(uri)
-          .asPresentation
-        contents.map(Right(_))
-      case (None, true) =>
-        logger.info("Payload in body and > auditing message limit")
-        (for {
-          parseResults <- FieldParsingService.getAdditionalFields(auditType.messageType, request.body).asPresentation
-          keyValuePairs = parseResults.collect {
-            case Right(pair) => pair
-          }
-          objSummary <- objectStoreService.putFile(fileId(), request.body).asPresentation
-        } yield ObjectSummaryWithFields(objSummary, keyValuePairs)).map {
-          summaryWithFields => Left(summaryWithFields)
+    if (exceedsLimit) {
+      logger.info("Payload in body and > auditing message limit")
+      (for {
+        parseResults <- FieldParsingService.getAdditionalFields(auditType.messageType, request.body).asPresentation
+        keyValuePairs = parseResults.collect {
+          case Right(pair) => pair
         }
-      case (Some(uri), true) =>
-        logger.info("Payload in object store and > auditing message limit")
-        for {
-          contents     <- objectStoreService.getContents(uri).asPresentation
-          parseResults <- FieldParsingService.getAdditionalFields(auditType.messageType, request.body).asPresentation
-          keyValuePairs = parseResults.collect {
-            case Right(pair) => pair
-          }
-          objSummary <- objectStoreService.putFile(fileId(), contents).asPresentation
-        } yield Left(ObjectSummaryWithFields(objSummary, keyValuePairs))
+        objSummary <- objectStoreService.putFile(fileId(), request.body).asPresentation
+      } yield ObjectSummaryWithFields(objSummary, keyValuePairs)).map {
+        summaryWithFields => Left(summaryWithFields)
+      }
+    } else {
+      logger.info("Payload in body and < auditing message limit")
```

Comment:
Are we keeping the logger.info until we're comfortable it's all working fine?

Reply:
For the moment, yeah -- I need to retest on QA once platops merge an internal auth change for me.